backend/TraderX/src/main/java/cmpe451/group6/rest/equipment/service/EquipmentUpdateService.java

```diff
@@ -57,7 +57,45 @@ private void initBase(){
         equipment.setCurrentValue(1); // since this is the base
         equipment.setLastUpdated(new Date());
 
-        equipmentRepsitory.save(equipment);
+        EquipmentRepository.save(equipment);
+    }
+
+    // For the ease of development only. Do not use on deployment
+    public void initMock(){
```

Comment:
Should we keep it when merging to backend-dev

Reply:
See reply: https://github.com/bounswe/bounswe2019group6/pull/178#discussion_r346690029